edb/server/pgcluster.py

```diff
@@ -71,14 +74,62 @@ class ClusterError(Exception):
     pass
 
 
+class BackendCapabilities(enum.IntFlag):
+
+    NONE = 0
+    #: Whether CREATE ROLE .. SUPERUSER is allowed
+    SUPERUSER_ACCESS = 1 << 0
+    #: Whether reading PostgreSQL configuration files
+    #: via pg_file_settings is allowed
+    CONFIGFILE_ACCESS = 1 << 1
+    #: Whether the PostgreSQL server supports the C.UTF-8 locale
+    C_UTF8_LOCALE = 1 << 2
+
+    ALL = SUPERUSER_ACCESS | CONFIGFILE_ACCESS | C_UTF8_LOCALE
```

Comment:
I suggest to not include computed flags into the enum. Including it makes it a proper member, making it show up when you iterate over individual flags etc. It'd just rename to `ALL_BACKEND_CAPABILITIES`